src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/UnitExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Xamarin.Platform
+{
+	public static class UnitExtensions
+	{
+		public static float EmCoefficient = 0.0624f;
+
+		public static float ToEm(this double pt)
+		{
+			return (float)pt * EmCoefficient; //Coefficient for converting Pt to Em
```

Comment:
This ... can't be right. The value for em is dependent on the font size. And even if it weren't, this assumes that we're using something like a 16.025pt font.

Reply:
Looks like this came straight out of our existing code; I'll open up an issue for it, and we'll deal with it separately.

Reply:
https://github.com/xamarin/Xamarin.Forms/issues/13859